realm/realm-library/src/androidTest/java/io/realm/RealmResultsTests.java

```diff
@@ -1765,36 +1765,40 @@ public void asJSON() throws JSONException {
         String json = all.asJSON();
         final String expectedJSON = "[\n" +
                 "    {\n" +
+                "        \"_key\":100," +
                 "        \"columnString\": \"alltypes1\",\n" +
                 "        \"columnLong\": 1337,\n" +
                 "        \"columnFloat\": 3.1400001,\n" +
                 "        \"columnDouble\": 0.89122999999999997,\n" +
                 "        \"columnBoolean\": false,\n" +
                 "        \"columnDate\": \"" + now + "\",\n" +
-                "        \"columnBinary\": \"010203\",\n" +
+                "        \"columnBinary\": \"AQID\",\n" +
```

Comment:
No idea, it concerned me a bit as well and I meant to ask Core about it

Reply:
No idea, but it concerned me a bit as well. I'll ask Core about it

Reply:
Binary data is now Base64 encoded apparently. I'll add a mention in the changelog. Normally it would be considered a breaking change as you removed the `@Beta` label on the `asJson()` method, but since it is also mentioned that the method is only for debugging it is probably acceptable? WDYT?